.github/workflows/cross-version-testing.md

```diff
@@ -112,6 +112,51 @@ the previous section:
 | 1.0.1         | ✅     | The latest micro version of `1.0`                  |
 | 1.1.dev       | ✅     | The version installed by `install_dev`             |
 
+## Why do we run tests against development versions?
+
+In cross-version testing, we daily run tests against not only publicly available versions but also
```

Comment:
"... we run daily tests against both publicly available and pre-release development versions for all dependent libraries that are used by MLflow. This section explains why."